bin/fetch-configlet

```diff
@@ -1,16 +1,26 @@
 #!/usr/bin/env python3.7
+from datetime import datetime, timezone
 import io
-import tarfile
+import json
 import platform
-import requests
 import sys
+import tarfile
+import time
+from urllib.request import urlopen
 
 
 def download_and_extract(url):
-    resp = requests.get(url)
-    print('extracting...')
-    file = tarfile.open(fileobj=io.BytesIO(resp.content), mode='r:gz')
-    file.extractall(path='bin/')
+    with urlopen(url) as request:
+        status = request.status
+        if status >= 400:
+            print(f"Unexpected {status} status from download server.")
+            time.sleep(5)
```

Comment:
This magic number 5 everywhere is going to be a pain if we ever decide to shorten or lengthen the backoff globally. Maybe a global `BACKOFF_TIME`?

Reply:
Good idea.